modules/apps/frontend-js/frontend-js-react-web/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+	"compilerOptions": {
+		"allowSyntheticDefaultImports": true,
+		"baseUrl": ".",
+		"module": "es6",
+		"moduleResolution": "node",
+		"outDir": "./build/node/packageRunBuild/dist/",
+		"paths": {
+			"@liferay/frontend-js-state-web": ["../frontend-js-state-web/*"]
+		},
```

Comment:
This is obviously the sort of thing that `@liferay/npm-scripts` could help us wire up automatically, depending on how the project references thing works out.